.github/workflows/deploy_prod.yml

```diff
@@ -71,6 +71,7 @@ jobs:
           SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
           APP_BUILD_VERSION: ${{ github.run_number }}
           APP_BINARY_VERSION: ${{steps.latestBinaryVersion.outputs.version}}
```

Comment:
ohnu, i forgot to add the spaces, can help add here too :p

Reply:
done!